.github/workflows/ci.yaml

```diff
@@ -65,6 +65,8 @@ jobs:
         run: |
           # Work around runtime failures on the GH Actions runner
           dotnet nuget locals all --clear
+          dotnet nuget add source https://api.nuget.org/v3/index.json -n nuget.org
+          dotnet restore
```

Comment:
Adding the source above appears to be the key fix. The restore should be run automatically as part of `dotnet build`, so isn't this line redundant?

Reply:
Most likely. I remove this. Can I remove ``` dotnet nuget locals all --clear``` ?

Reply:
I don't remember whether that was working around the same issue that the source fixes or something else. Try it, and see if the build fails?